### PR TITLE
[Fix-issue:2201] “Use Python for…” section contains outdated links

### PR DESCRIPTION
**Pull request by tiidadavena**
**Description**

This PR aims to solve the issue #2201.

**Solution**
 
It was created a new fixture to update the `Boxes` model.

**Screenshots**
image

### CODE REVIEW

File: fixtures/boxes.json [hunk 1]
```diff
@@ -1,758 +1,884 @@
...
-      "label": "supernav-python-about",
-      "content": "\r\n                            <li class=\"tier-2 super-navigation\">\r\n                                <h4>Python is a programming language that lets you work more quickly and integrate your systems more effectively.</h4>\r\n                                <p>You can learn to use Python and see almost immediate gains in productivity and lower maintenance costs.  <a href=\"/about\">Learn more about Python</a>.\r\n                            </li>",
-      "content_markup_type": "html",
-      "_content_rendered": "\r\n                            <li class=\"tier-2 super-navigation\">\r\n                                <h4>Python is a programming language that lets you work more quickly and integrate your systems more effectively.</h4>\r\n                                <p>You can learn to use Python and see almost immediate gains in productivity and lower maintenance costs.  <a href=\"/about\">Learn more about Python</a>.\r\n                            </li>"
+        "created": "2013-03-11T22:38:14.817Z",
+        "updated": "2014-06-25T19:01:06.268Z",
+        "creator": null,
```
Comment (eamanu) [inline fixtures/boxes.json:8]: creator and last_modified_by are new keys and set as null, they are necessary?

**Comment by eamanu** [comment]
Hi, sorry if I'm wrong, I don't have too much experience in this project, but the diff of new changes are so big, I would expect see only the links changed.

**Comment by tiidadavena** [comment]
> Hi, sorry if I'm wrong, I don't have too much experience in this project, but the diff of new changes are so big, I would expect see only the links changed.

I guess it was caused because the `boxes.json` has too many lines and other some minor updates were also applied after running the latest migration.

cc @Mariatta

**Comment by ambv** [comment]
The amount of unrelated changes here makes this impossible to review. Please open a new PR with just the relevant lines touched.

**Pull request closed without merging**